### PR TITLE
Add ability to exclude users from pick

### PR DESCRIPTION
**Pull request by NedReidSoftwire**
- I haven't tested this code
- It should hopefully add the ability to exclude users:
  - probably 🤷‍♀️

### CODE REVIEW

**Comment by aleyipsoftwire** [comment]
Is the purpose of this to be able to explicitly exclude a user from a particular pick?

**Comment by NedReidSoftwire** [comment]
> Is the purpose of this to be able to explicitly exclude a user from a particular pick?

That is correct 😃  If for example it's a QA ticket and someone specifically has already done the PR for it

File: README.md [hunk 1]
```diff
@@ -12,6 +12,15 @@
 | `show <team name>` | Show team and manage members. | `@Pickir show a-team` |
 | `stats` | This is a WIP feature.<br>Show statistics for the current channel.  | `@Pickir stats` |
 
+## Extra flags for picks
+Note that one must currently call the exclude tag after the team tag, if both are provided.
+
+| flag                                      | Description                  |
+|-------------------------------------------|------------------------------|
+| `--team <team name>`, (`-t`)              | Only pick from specific team |
```
Comment (alexttyip) [inline README.md:20]: Nice spot, totally forgot to add this to the readme

**Review by alexttyip — APPROVED** [review]
Looks good, I'll add you to the test channel on slack so you can test it

**Pull request merged**